include/identity.php

```diff
@@ -663,6 +668,10 @@ function advanced_profile(App $a) {
 			$profile['age'] = array( t('Age:'), $age );
 		}
 
+		if ($age = age($a->profile['dob'], $a->profile['timezone'], '')) {
```

Comment:
Shouldn't it be ```==``` instead of ```=```? But even then I don't really understand the meaning.